src/Exception/ApiException.php

```diff
@@ -50,7 +50,7 @@ public function __construct(GuzzleRequestException $previous, $message = '')
 
         if ($this->previous->hasResponse()) {
             $this->response = $this->previous->getResponse();
-            $this->requestId = $this->response->getHeader('X-Contentful-Request-Id')[0];
+            $this->requestId = $this->response->getHeaderLine('X-Contentful-Request-Id');
```

Comment:
To be honest, making some weird workarounds for possible bugs that may not ever happen for a very specific implementation where the worst thing that could happen is to actually report the ID twice (should such a bug ever arise), well, it seems a bit silly to me...